src/main/drivers/mco.c

```diff
@@ -89,6 +89,8 @@ void mcoConfigure(MCODevice_e device, const mcoConfig_t *config)
         IOConfigGPIOAF(io, IO_CONFIG(GPIO_Mode_AF, GPIO_Speed_50MHz, GPIO_OType_PP, GPIO_PuPd_NOPULL), GPIO_AF_MCO);
 #endif
         break;
+    default:
```

Comment:
This default will only mask problem. Without it, compiler will generate warning for missing case (when MCODevice_e  is extended)

Reply:
Deleted